pkg/server/message.go

```diff
@@ -67,88 +67,88 @@ func (t *TranslateServiceServer) CreateMessages(
 		return nil, status.Errorf(codes.InvalidArgument, err.Error())
 	}
 
-	msgs, err := t.repo.LoadMessages(ctx, params.serviceID,
-		repo.LoadMessagesOpts{FilterLanguages: []language.Tag{params.messages.Language}})
+	translation, err := t.repo.LoadTranslation(ctx, params.serviceID,
```

Comment:
For now when we save, we save one translation e.g. one language, but when we load we get multiple translations 
```suggestion
	translations, err := t.repo.LoadTranslations(ctx, params.serviceID,
```

Reply:
even when we expect one translation, it seems odd to see check like this

```go
len(translation) > 0
```